internal/feature/flags.go

```diff
@@ -0,0 +1,7 @@
+package feature
+
+// Flags contains the values of the feature flags.
+type Flags struct {
+	// EnableUpgradeAgent is a boolean flag to enable or disable the upgrade agent feature:
+	EnableUpgradeAgent bool `envconfig:"ENABLE_UPGRADE_AGENT" default:"true"`
```

Comment:
We are going to move all our flags here?

Reply:
I suggest to do so, yes.